controllers/v1/observationsController.js

```diff
@@ -847,10 +847,11 @@ module.exports = class Observations extends Abstract {
                 }
 
                 let assessment = {};
-
+                
                 assessment.name = solutionDocument.name;
                 assessment.description = solutionDocument.description;
                 assessment.externalId = solutionDocument.externalId;
+                assessment.pageHeading = solutionDocument.pageHeading;
```

Comment:
@rakeshSgr - Might as well do in v2 and also in assessments controller as well

Reply:
resolved